iothub/device/src/DirectMethod/DirectMethodRequest.cs

```diff
@@ -82,12 +83,6 @@ public DirectMethodRequest(string methodName)
         [JsonIgnore]
         public string RequestId { get; protected internal set; }
 
-        /// <summary>
-        /// The JSON payload in JRaw type.
-        /// </summary>
-        [JsonProperty("payload", NullValueHandling = NullValueHandling.Include)]
-        protected internal JRaw JsonPayload { get; set; }
```

Comment:
We didn't seem to use this from the client, nor even use the JsonProperty attributes.